joomla/joomla-cms
# [4.0] WebAsset for layouts

### Summary of Changes
This replaces htmlhelper to webaset for layouts

Updated layouts:
```
joomla/html/batch/item
joomla/html/batch/language
joomla/html/batch/adminlanguage
joomla/form/field/calendar
joomla/form/field/color/slider
joomla/form/field/password
joomla/form/renderfield
joomla/toolbar/containeropen
joomla/searchtools/default/filters
joomla/edit/frontediting_modules
joomla/content/options_default
plugins/system/webauthn/manage
```


### Testing Instructions
Apply patch. Run `npm install`
Click around, check calendar, template style colors, batch,  fields "showon",
all should work as before


### Expected result
All work


### Actual result
All work


### Documentation Changes Required
nope

ref #22435
